src/docbuilder/metadata.rs

```diff
@@ -111,6 +120,8 @@ impl Metadata {
                         .and_then(|v| v.as_bool()).unwrap_or(metadata.all_features);
                     metadata.default_target = table.get("default-target")
                         .and_then(|v| v.as_str()).map(|v| v.to_owned());
+                    metadata.extra_targets = table.get("extra-targets").and_then(|f| f.as_array())
+                        .and_then(|f| f.iter().map(|v| v.as_str().map(|v| v.to_owned())).collect());
```

Comment:
You don't have to do it in this PR, but it would be nice to just use `#[derive(Deserialize)]`.